awscdk-minecraft/tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+"""Reusable fixtures for the tests."""
+import os  # noqa: D100 C0114
+
+import pytest
+from aws_cdk import Environment
+
+
+@pytest.fixture(scope="module")
+def dev_env():  # noqa: D103
+    return Environment(account=os.environ["AWS_ACCOUNT_ID"], region=os.environ["AWS_REGION"])
```

Comment:
@phitoduck I used the `app.py` file to figure this out, but noticed that one env variable uses `os.environ` and the other uses `os.getenv`, and the latter was tripping up my ability to run pytest for some reason, so I changed both to use `os.environ` and used `export` to set the relevant environment variables in my shell.